packages/plugins/plugin-onetrust/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "@segment/analytics-react-native-plugin-onetrust",
+  "version": "0.0.0",
```

Comment:
What version will this end up getting published as?

Reply:
Depends on what we decide while running `yarn version` command - whether we want to release it as `1.0.0` or `1.0.0-rc` or `1.0.0-alpha` etc. I don't have knowledge on what our stance is around shipping this entire feature.

Reply:
Seems like for Kotlin it is being released as 1.0.x right now
I think if we change the commit message to `feat: consent-management` ti will release it properly